lib/tree/peer.go

```diff
@@ -0,0 +1,74 @@
+package tree
+
+import (
+	"github.com/dedis/cothority/lib/hashid"
+	"github.com/dedis/cothority/lib/network"
+	"github.com/dedis/crypto/abstract"
+	"strconv"
+)
+
+// Peer represents a Cothority-member
+type Peer struct {
+	// The hostname of the peer
+	Name string
+	// The port-number
+	Port int
+	// The public-key
+	PubKey abstract.Point
+	// The private-key - mostly for ourselves
+	PrivKey abstract.Secret
+	// A network-connection - if already set up
+	Conn network.Conn
+}
+
+// PeerList regroup a number of peers in a list. One peer can be
+// member of more than one PeerList.
+type PeerList struct {
+	// A list of all peers that are part of this list
+	Peers []*Peer
+	// The hash-id of this list
+	Hash hashid.HashId
+	// The suite used in this list
+	Suite abstract.Suite
```

Comment:
Following our discussion, I think suite should not be here. Should it not be higher up, in the definition of what application we want to run(stamp,bitcosi etc) ?